docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -774,6 +774,23 @@ of these options, because they would then not be verified against actual
 trusted CA certificates.
 
 
+IPv4/IPv6 Connections
+---------------------
+
+When a client is constructed with ``localhost`` as the host name, it will
+attempt an IPv4 connection only (i.e. if ``localhost`` resolves to
+``127.0.0.1`` and ``::1``, the driver will only try to connect to
+``127.0.0.1``).
+
+When a client is constructed with hostnames other than ``localhost``, it will
+attempt both IPv4 and IPv6 connections depending on the addresses that the
+hostnames resolve to. The driver respects the order in which ``getaddrinfo``
+returns the addresses, and will attempt to connect to them sequentially.
+The first successful connection will be used.
+
+The driver does not currently implement the Happy Eyeballs algorithm.
```

Comment:
Is it worth linking "Happy Eyeballs" to https://tools.ietf.org/html/rfc6555 or https://en.wikipedia.org/wiki/Happy_Eyeballs?

Reply:
I capitalized H and E to make it clear that "happy eyeballs" is the name of the algorithm. If confusion remains this can also explicitly mention RFC 6555.